src/promptflow/promptflow/_sdk/_orm/experiment.py

```diff
@@ -55,6 +56,8 @@ def dump(self) -> None:
                 if "UNIQUE constraint failed" not in str(e):
                     raise
                 raise ExperimentExistsError(f"Experiment name {self.name!r} already exists.")
+            except Exception as e:
+                raise UserErrorException(target=ErrorTarget.CONTROL_PLANE_SDK, message=str(e), error=e)
```

Comment:
Let's raise more concrete Error instead of the  base UserErrorException

Reply:
Actually, I am not sure what type e will throw. I assume there are many types that I cannot capture one by one.
When I use `UserErrorException `and` error=e`, it sets the `error category` to `UserError ` instead of `UNKNOW`, and the `error type` to `error. _ name__`
For example, in the following error, the `error type` will be set to `OperationalError`.
![image](https://github.com/microsoft/promptflow/assets/23182548/1aac8318-a467-4a81-b073-aff1eb180038)

Reply:
According to offline discussions, we will further optimize the details for the next PR, and merge the current PR first